app/views/devise/registrations/edit.html.erb

```diff
@@ -30,6 +30,57 @@
     <%= f.label :current_password %> <i>(we need your current password to confirm your changes)</i><br />
     <%= f.password_field :current_password, autocomplete: "current-password" %>
   </div>
+  <br />
+  <!--Personal Info from new user-->
+  <div class="col-8">
+  <div class="card text-white bg-info mb-3" style="max-width: 50rem;">
+    <div class="card-header">User Information</div>
+    <div class="card-body">  
+      <h5 class="card-title">User</h5>
+        <div>
+             <%= f.label :picture %>
+             <%= f.text_field :picture, autofocus: true %>
```

Comment:
If the idea here is for a user to upload a picture, then you'll need to use something other than `f.text_field` here.

For a helpful tutorial on how to allow users to upload images files, check out https://www.railstutorial.org/book/user_microposts#sec-micropost_images.

Reply:
This still needs to be updated so that it's possible to upload a picture.